docs/nodes/CAD/embed mesh.rst

```diff
@@ -0,0 +1,49 @@
+Embed Mesh
+===============
+
+.. image:: https://user-images.githubusercontent.com/74725748/204120088-03599881-59d5-4f6c-b7de-3b90b1f620cc.png
+
+Functionality
+-------------
+Embed mesh B on mesh A into mesh A, and in simple terms, you will get the structure of mesh B on mesh A
+
+
+Inputs
+------
+
+- **VertsA** - vertices of mesh A
+- **EdgesA** - edges of mesh A
+- **FacesA** - faces of mesh A
+- **VertsB** - vertices of mesh B
+- **EdgesB** - edges of mesh B
+- **FacesB** - faces of mesh B
+- **Index**  - Specifies on which face of mesh A each point of mesh B is on
```

Comment:
Please, try to use trivial **Donor** and **Recipient** naming as for similar nodes presents

Reply:
Which name do you think is better? This is the node of interest
![image](https://user-images.githubusercontent.com/74725748/205176654-d7f47f3f-ecf8-470e-8b2a-065ca2ae35ff.png)

Reply:
not sure